js/script.js

```diff
@@ -38,7 +38,7 @@ placeable[4] = height/length
 placeable[5] = mirror flipped horizontal rotation
 */
 
-var placeable = [
+var placeable = factorioEntityObjectArrayFromMultiArray([

```

Comment:
instead of writing a function to turn these arrays into Entity objects, couldn't we just create Entity objects here?